rollup.config.js

```diff
@@ -0,0 +1,22 @@
+// Look into using rollup.js built in "watch" instead of nodemon
+
+const path = require('path');
+const fs = require('fs');
+
+const outConfig = []
+
+const dirItems = fs.readdirSync(`${__dirname}${path.sep}src`, { withFileTypes: true });
+for (let i = 0; i < dirItems.length; i++) {
+  const item = dirItems[i];
+  if (item.name.endsWith('.js') && !item.name.endsWith('.test.js')) {
+    outConfig.push({
+      input: `${__dirname}${path.sep}src${path.sep}${item.name}`,
+      output: [{
+        file: `${__dirname}${path.sep}dist${path.sep}${item.name}`,
+        format: 'es',
+      }],
+    });
+  }
+}
+
+export default outConfig;
```

Comment:
The rollup config requires an entry for each file to be transformed. In this case we transform every file that is a direct descent of the src folder that isn't a test. If we decide in the future to change the approach then this file will need to be updated to reflect the changes.